tools/apiview/emitters/typespec-apiview/src/apiview.ts

```diff
@@ -73,10 +75,10 @@ export class ApiView {
   typeDeclarations = new Set<string>();
   includeGlobalNamespace: boolean;
 
-  constructor(name: string, packageName: string, packageVersion?: string, includeGlobalNamespace?: boolean) {
+  constructor(name: string, packageName: string, includeGlobalNamespace?: boolean) {
     this.name = name;
     this.packageName = packageName;
-    this.packageVersion = packageVersion ?? "";
+    this.packageVersion = "ALL";
```

Comment:
Does this mean TypeSpec API review won't have any version info?

Reply:
It only ever contained the tool version. The package version isn't written anywhere and that has been true the whole time since the Powershell script that invoked it never supplied a version.